defaults/templates/drupal.settings.build.php

```diff
@@ -24,6 +24,9 @@
 $settings['file_public_path'] = '${drupal.site.settings.file_public_path}';
 $settings['file_private_path'] = '${drupal.site.settings.file_private_path}';
 
+// Enable/disable config_split configurations.
```

Comment:
I might recommend changing this to the following (or adding):
> To simulate other config split environments, change `development` to `staging` or `production`, then run `drush cr && drush cim -y`

Reply:
Done!